content/en/user-guide/web-application/accounts/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Accounts"
+tags: [] 
+weight: 2
+description: >
+  LocalStack accounts are the entry point to our web application, the pro CLI, and more!
+---
+
+A LocalStack account is a requirement to get your LocalStack licenses, which allow you to use advanced features of LocalStack, such as LocalStack Pro services, [Stack Insights]({{< ref "stack-insights" >}}), [Cloud Pods]({{< ref "cloud-pods" >}}) and more!<br><br>
+To create an account for LocalStack, please visit our [web application](https://app.localstack.cloud/sign-up).<br>
+When signing up for an account, you can either do so by providing an email address, or by signing up with one of our social identity providers (such as GitHub).
+
+<img src="signup.PNG" width="550px" alt="Sign Up Form">
+
+### Individual Account / Company Account
+After successfully signing up for a LocalStack, you will end up with an <b>individual account</b>.
+These accounts are intended for non company entities, such as sole developers.<br>
+Within the [profile](https://app.localstack.cloud/account) section of your account, you will find a toggle to switch to a <b>company account</b> instead.
+This will allow you to add more information about your company, such as `Company Name`, `Country` and `Tax ID`, which should be set if you are acting on behalf of a company.
```

Comment:
```suggestion
This will allow you to add more information about your organization, such as `Company Name`, `Country` and `Tax ID`, which should be set if you are acting on behalf of a company.
```